components/02-molecules/modal/_modal.scss

```diff
@@ -0,0 +1,95 @@
+// Base modal styles. Additional classes accommodate for custom app implementations.
+.modal__toggle {
+  cursor: pointer;
+}
+
+.modal__pane {
+  background-color: clr(background-section);
+  color: clr(text);
+  left: 0;
+  opacity: 0;
+  padding: $space;
+  position: fixed;
+  top: 50%;
+  transform: translate(0%, -50%);
+  transition: opacity 300ms;
+  visibility: hidden;
+  width: 100vw;
+  z-index: 1000;
+
+  @include large {
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: $small;
+  }
+
+  .modal--active & {
+    max-height: 100vh;
+    opacity: 1;
+    visibility: visible;
+  }
+}
+
+.modal__title {
+  @include wrapper($container-max-width: $small);
+
+  margin-bottom: $space;
+}
+
+.modal__content {
+  @include wrapper($container-max-width: $small);
+
+  > * {
```

Comment:
I'm a little confused by this. Let me know if I'm getting this wrong: so all the modal content gets piped through the `paragraph` component, which means an example could be:
```
<p>
  <div>content here
    <span>here
      <h2>oh a heading</h2>
    </span>
  </div>
</p>
```
Won't the `p` always be the last child?